src/app/services/routes.ts

```diff
@@ -48,13 +48,6 @@ export const routes = () => {
             module: 'services',
           },
           component: () => import('@/app/services/views/ServiceListView.vue'),
-          children: [
-            {
-              path: ':service',
-              name: 'service-summary-view',
-              component: () => import('@/app/services/views/ServiceSummaryView.vue'),
-            },
-          ],
```

Comment:
I noticed we don't use `ServiceSummaryView` as clicking on a Service row takes you straight to a detail page instead of a summary drawer. So instead of updating the ResourceCodeBlock in a file we don't use, I just deleted the file (plus a related test, plus the route configuration you can see here)